test/functional/platform/Document.spec.js

```diff
@@ -1,5 +1,162 @@
-describe('Platform', () => {
+const {
+  PrivateKey,
+  PublicKey,
+} = require('@dashevo/dashcore-lib');
+
+const DAPIClient = require('@dashevo/dapi-client');
+const DashPlatformProtocol = require('@dashevo/dpp');
+
+const GrpcErrorCodes = require('@dashevo/grpc-common/lib/server/error/GrpcErrorCodes');
+
+const getDataContractFixture = require(
+  '@dashevo/dpp/lib/test/fixtures/getDataContractFixture',
+);
+
+const fundAddress = require('../../../lib/test/fundAddress');
+const createOutPointTxFactory = require('../../../lib/test/createOutPointTxFactory');
+
+describe('Platform', function platform() {
+  this.timeout(950000);
+
+  let dapiClient;
+  let dpp;
+  let identityPrivateKey;
+  let dataContract;
+  let publicKeyId;
+  let identity;
+  let document;
+
+  before(async () => {
```

Comment:
Instead of creating SDK in bootstrap, let's create a function `getClientWithFundedWallet`